infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars

```diff
@@ -86,55 +86,55 @@ cloud_run_config = {
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
```

Comment:
Endpoint needs updating to `https://australia-southeast1-docker.pkg.dev`

Reply:
done.